### ISSUE TITLE
Glossary - Search Labels [RDI 2.23]

### ISSUE THREAD

**Issue by MeD-Owner** [MeD-DMC/Adapt-accessibilityfixes#82]
I'm not sure what thecheckbox does since it does not have a visible label. Plus it seems to affect the output of the search. 

When testing with JAWS, the two items are announced as: 

Edit type and text 

Accessibility, checkbox not checked (or checkbox checked) 

Note that the type and text box has no label so I do not know what should be written here or why. Visually we can see the magnifier glass, but it is not announced. 

I have no idea what the checkbox does. Note that when the focus is on the checkbox and JAWS will announce “"word” checkbox not checked”. “word” represents the first item displayed in the results list. This demonstrates another issue with the checkbox – that it has no label. JAWS seems to take the first result in the list as the label for the checkbox.

**Comment by GGD4V** [comment]
Tons of labels are avail in general settings that weren't filled in RDI. When you put all the labels in general settings, there is a clear association between the checkbox and its label.

**Comment by MeD-Owner** [comment]
adding labels seem to fix it, but needs re-testing

**Comment by GGD4V** [comment]
To put in best practices.
